src/store/modules/tester/codeToPos.js

```diff
@@ -0,0 +1,213 @@
+export const ANSI = [
```

Comment:
I think they differ by 2 positions. Let's just use ANSI as the base and slice ISO

Reply:
I don't agree with this. Slicing would be a real mess to read. Like having the base (ANSI) layout and manipulate it to set it to ISO or JIS? Not maintainable IMO.